docs/dotnet/dotnet-aspnetcore-metrics.md

```diff
@@ -0,0 +1,226 @@
+| `aspnetcore.routing.match_status` | string | Match result - success or failure | `success`; `failure` | Required |
+| [`http.route`](../attributes-registry/http.md) | string | The matched route (path template in the format used by the respective server framework). See note below [1] | `/users/:userID?`; `{controller}/{action}/{id?}` | Conditionally Required: if and only if a route was successfully matched. |
+
+**[1]:** MUST NOT be populated when this is not supported by the HTTP server framework as the route attribute should have low-cardinality and the URI path can NOT substitute it.
+SHOULD include the [application root](/docs/http/http-spans.md#http-server-definitions) if there is one.
+
+`aspnetcore.routing.match_status` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
+
+| Value  | Description |
+|---|---|
+| `success` | Match succeeded |
+| `failure` | Match failed |
+<!-- endsemconv -->
+
+## Exception metrics
+
+Metrics reported by `Microsoft.AspNetCore.Diagnostics` meter.
+
+### Metric: `aspnetcore.diagnostics.exceptions`
+
+<!-- semconv metric.aspnetcore.diagnostics.exceptions(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `aspnetcore.diagnostics.exceptions` | Counter | `{exception}` | Number of exceptions caught by exception handling middleware. [1] |
+
+**[1]:** Meter name: `Microsoft.AspNetCore.Diagnostics`; Added in: ASP.NET Core 8.0
+<!-- endsemconv -->
+
+<!-- semconv metric.aspnetcore.diagnostics.exceptions(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `aspnetcore.diagnostics.exception.result` | string | ASP.NET Core exception middleware handling result | `handled`; `unhandled` | Required |
+| `aspnetcore.diagnostics.handler.type` | string | Full type name of the [`IExceptionHandler`](https://learn.microsoft.com/dotnet/api/microsoft.aspnetcore.diagnostics.iexceptionhandler) implementation that handled the exception. | `Contoso.MyHandler` | Conditionally Required: [1] |
+| `error.type` | string | The full name of exception type. [2] | `System.OperationCanceledException`; `Contoso.MyException` | Required |
+
+**[1]:** if and only if the exception was handled by this handler.
+
+**[2]:** The `error.type` SHOULD be predictable and SHOULD have low cardinality.
+Instrumentations SHOULD document the list of errors they report.
+
+The cardinality of `error.type` within one instrumentation library SHOULD be low, but
+telemetry consumers that aggregate data from multiple instrumentation libraries and applications
+should be prepared for `error.type` to have high cardinality at query time, when no
+additional filters are applied.
```

Comment:
It does not really matter - markdown rendering takes care of better line breaks as you can see here 

https://github.com/open-telemetry/semantic-conventions/blob/323fd0b3c15e7c725f403ab8b0dcecae1ad07109/docs/dotnet/dotnet-aspnetcore-metrics.md

The challenge here is that this text is written down in yaml files and auto-generated into markdown. Enforcing one-line in MDs (while also sometimes wanting to enforce multi-lines, lists, etc) would be too hard.

Reply:
> It does not really matter - markdown rendering takes care of better line breaks as you can see here
> 
> https://github.com/open-telemetry/semantic-conventions/blob/323fd0b3c15e7c725f403ab8b0dcecae1ad07109/docs/dotnet/dotnet-aspnetcore-metrics.md
> 
> The challenge here is that this text is written down in yaml files and auto-generated into markdown. Enforcing one-line in MDs (while also sometimes wanting to enforce multi-lines, lists, etc) would be too hard.

That explains it. We want to make all the changes at the source and minimize generated file changes.

Reply:
> The challenge here is that this text is written down in yaml files and auto-generated into markdown.

Where is the yaml file that generates this?

Reply:
> > The challenge here is that this text is written down in yaml files and auto-generated into markdown.
> 
> Where is the yaml file that generates this?

In this case, it's generated from this: 

https://github.com/open-telemetry/semantic-conventions/blob/af302dd70567eeceb4bfb1df479e3d2580738a91/model/error.yaml#L22

But we don't need to change the source since readers viewing rendered markdown would not notice the difference, but we (people who write in markdown) will have a harder time.